lbz/jwt_utils.py

```diff
@@ -26,16 +26,16 @@ def get_matching_jwk(auth_jwt_token: str) -> dict:
             if key["kid"] == kid_from_jwt_header:
                 return key
 
-        logger.error(f"Required key not found in configuration.")
```

Comment:
FYI: I don't think it is worth having error logs for this situation (client error).

Reply:
I'd also adjust the message a bit:

`"The key with id={kid_from_jwt_header} was not found in the environment variable."`